src/Pages/Profile/MemberView/profile-modifier.css

```
@@ -45,5 +46,54 @@
   height: 35%;
   top: 25%;
   left: 58%;
-  z-index: 1;
+  z-index: 2;
+}
+
+#printing-request-button {
+  margin-top: 10px;
+  margin-left: 10%;
+}
+
+@media screen and (max-width: 1200px) {
+  #profileLogo {
+    height: 15%;
+    left: 60%;
+  }
+
+  #innerTextTop{
+    font-size: 20px;
+  }
+
+  #profileBox {
```

Please follow css naming conventions from now on, no camelcase
```suggestion
  #profile-box {
```